plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/handler/OpenInWebConsoleHandler.java

```diff
@@ -90,6 +91,8 @@ private String getWebConsoleUrl(Connection connection, IResource resource) {
 				url.append("/services/").append(resource.getName());
 			} else if (resource instanceof IImageStream) {
 				url.append("/images/").append(resource.getName());
+			} else if (resource instanceof IPersistentVolumeClaim) {
```

Comment:
I think this surrounding 'if' block should be refactored because of all the mostly same logic; maybe introduce a map